frame/merkle-mountain-range/src/mmr/storage.rs

```diff
@@ -93,7 +92,7 @@ where
 		}
 
 		sp_std::if_std! {
-			log::trace!("elems: {:?}", elems.iter().map(|elem| elem.hash()).collect::<Vec<_>>());
+			frame_support::log::trace!("elems: {:?}", elems.iter().map(|elem| elem.hash()).collect::<Vec<_>>());
```

Comment:
not sure why but okay :D

Reply:
It was giving warning given that we import it for the full file while it's only needed when we have `std`